darwin/dataset.py

```diff
@@ -157,8 +158,8 @@ def upload_file_to_s3(client: Client, file: requests.Response) -> str:
     end_point = response["postEndpoint"]
 
     s3_response = upload_to_s3(signature, end_point, file_path)
-    if not str(s3_response.status_code).startswith("2"):
-        process_response(s3_response)
+    # if not str(s3_response.status_code).startswith("2"):
+    #     process_response(s3_response)
```

Comment:
Since we currently don't have this `process_response` function implemented, I commented in these lines